go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/ipfs/go-unixfs v0.2.4
 	github.com/ipfs/go-verifcid v0.0.1
 	github.com/ipfs/interface-go-ipfs-core v0.4.0
-	github.com/ipld/go-car v0.1.0
+	github.com/ipld/go-car v0.1.1-0.20200429200904-c222d793c339
```

Comment:
Is there a release of go-car we can use here?

Reply:
No, this is what I was referring to in my original comment: go-car needs a bit more work for a release.

Reply:
@aschmahmann this is go-car master fwiw, so you are not depending on some random commit. I need to sneak in a particular fix in ( no API changes ), and that's another day or 2 away.

Reply:
So should I be merging this and then we can update go-car later or am I waiting to merge on a go-car release?

Reply:
Let's merge this now, I won't be ready in a reasonable timeframe.